docs/content/docs/custom-resource/autoscaler.md

```diff
@@ -37,7 +37,10 @@ Key benefits to the user:
 
 Job requirements:
  - The autoscaler currently only works with the latest [Flink 1.17](https://hub.docker.com/_/flink) or after backporting the following fixes to your 1.15/1.16 Flink image
-   - [Job vertex parallelism overrides](https://github.com/apache/flink/commit/23ce2281a0bb4047c64def9af7ddd5f19d88e2a9) (must have)
+   - Job vertex parallelism overrides (must have)
+     - [Add option to override job vertex parallelisms during job submission](https://github.com/apache/flink/commit/23ce2281a0bb4047c64def9af7ddd5f19d88e2a9)
+     - [Change ForwardPartitioner to RebalancePartitioner on parallelism changes](https://github.com/apache/flink/commit/61d6e78e1f6dff093e005c834e1a008f146c02b9)
```

Comment:
As @gyfora  said at [FLINK-30213](https://issues.apache.org/jira/browse/FLINK-30213?focusedCommentId=17653597&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-17653597), the PR is merged by 5 commits. So it's better to provide the PR link instead of commit link.

```suggestion
     - [Change ForwardPartitioner to RebalancePartitioner on parallelism changes](https://github.com/apache/flink/pull/21443)
```

Reply:
Thanks @1996fanrui! I didn't notice it before, will fix it